package validation.interceptor.exception;

public class AuthException extends RuntimeException{
}
